srcpkgs/ncdc/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'ncdc'
+pkgname=ncdc
+version=1.20
+revision=1
+build_style=configure
+configure_args="--prefix=/usr --with-geoip"
+hostmakedepends="pkg-config glib-devel ncurses-devel sqlite-devel gnutls-devel geoip-devel"
```

Comment:
most if not all of the `-devel` packages schould be in `makedepends`

Reply:
I can see as minimum pkg-config and glib-devel in hostmakedepens in many templates

Reply:
Yes. glib-devel contains binaries that need to be executed on the host. If any of these are needed, it's correct to put it in `hostmakedepends`. pkg-config always goes there as it is a binary to be executed on the host. If none of the binaries are needed, it could also be enough to put `libglib-devel` in `makedepends`.